cmd/ttn-lw-cli/commands/gateways.go

```diff
@@ -471,6 +471,38 @@ var (
 			return io.Write(os.Stdout, config.OutputFormat, res)
 		},
 	}
+	gatewaysBatchConnectionStats = &cobra.Command{
+		Use:     "batch-get-connection-stats",
+		Aliases: []string{"batch-connection-stats", "batch-cnx-stats"},
+		Short:   "Get connection stats for a batch of gateways",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ids, err := cmd.Flags().GetStringSlice("gateway-ids")
+			if err != nil {
+				return err
+			}
+			var gtwIDs []*ttnpb.GatewayIdentifiers
+			for _, id := range ids {
+				gtwIDs = append(gtwIDs, &ttnpb.GatewayIdentifiers{
+					GatewayId: id,
+				})
+			}
+
+			gs, err := api.Dial(ctx, config.GatewayServerGRPCAddress)
+			if err != nil {
+				return err
+			}
+
+			res, err := ttnpb.NewGsClient(gs).BatchGetGatewayConnectionStats(ctx,
+				&ttnpb.BatchGetGatewayConnectionStatsRequest{
+					GatewayIds: gtwIDs,
+				})
+			if err != nil {
+				return err
+			}
+
+			return io.Write(os.Stdout, config.OutputFormat, res)
+		},
+	}
```

Comment:
Why not make this part of the existing `get-connection-stats`? We can just adjust the logic if more than one gateway ID is passed, right?

Reply:
Good one 👍